client/src/redux/store.js

```diff
@@ -1,11 +1,13 @@
 import { configureStore } from '@reduxjs/toolkit';
 import authReducer from './features/auth/authSlice';
+import userReducer from './features/auth/user';
 // Create the store, and export it for use in the application
 const store = configureStore({
   reducer: {
     // reducer name
     // reducer function
     auth: authReducer,
+    user: userReducer,
```

Comment:
We created the file set up for redux, why you are adding a new one?
You already have the authReducer and the authSlice, so just fill them